frontend/src/components/AboutMe/UserPaper.jsx

```
@@ -0,0 +1,119 @@
+            </button>
+            <button
+            onClick={() => {
+                console.log("paper:",paper)
+                handleAskU(paper)}}
+            className="relative inline-flex items-center gap-2 text-sm font-medium text-white bg-gradient-to-r from-indigo-500 via-purple-600 to-pink-500 px-4 py-2 rounded-xl shadow-lg hover:shadow-xl hover:scale-[1.02] transition-transform duration-200"
+            >
+            <Bot className="w-4 h-4 animate-pulse" />
+            <span className="tracking-wide">AskU</span>
+
+            {/* Online Indicator */}
+            <span className="absolute -top-1 -right-1 w-2.5 h-2.5 bg-green-400 rounded-full border-2 border-white animate-ping"></span>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)